File: HeroesApp/src/app/models/hero.ts
export class Hero {
    id: number;
    name: string;
    bio: string;
    image: string;
    firstAppearance: Date;
    homeId: number
    home: HeroHome;
}

export class HeroHome {
    id: number;
    name: string;
    image: string;
}

File: HeroesBackend/HeroesBackend.Entities/Hero.cs
using System;

namespace HeroesBackend.Entities
{
    public class Hero : BaseEntity
    {
        public string Name { get; set; }
        public string Bio { get; set; }
        public string Image { get; set; }
        public DateTime FirstAppearance { get; set; }
        public long HomeId { get; set; }
        public HeroHome Home { get; set; }
    }
}
